tests/unit/compiler/test_source_map.py

```diff
@@ -101,6 +101,15 @@ def update_foo():
     assert "fallback function" in list(error_map.values())
 
 
+def test_error_map_with_user_error():
+    code = """
+@external
+def foo():
+    raise "some error"
+    """
+    error_map = compile_code(code, output_formats=["source_map"])["source_map"]["error_map"]
+    assert "user revert with reason" in list(error_map.values())
```

Comment:
why can't we  check `in error_map.values()`?

Reply:
We can. `test_error_map` also wraps the values in list - should I change it there too?

Reply:
it's a dict view, i think we should. @charles-cooper do you remember the reason you did it this way?

Reply:
> it's a dict view, i think we should. @charles-cooper do you remember the reason you did it this way?

no, just habit i think